CredentialProvider/CredentialListPresenter.swift

```diff
@@ -24,7 +24,7 @@ class CredentialListPresenter {
     
     func filterCredentials(for searchText: String) {
         filteredCredentials = loginsData.filter { item in
-            item.0.serviceIdentifier.identifier.lowercased().contains(searchText.lowercased())
+            item.0.serviceIdentifier.identifier.titleFromHostname.lowercased().contains(searchText.lowercased()) || item.0.user.lowercased().contains(searchText.lowercased())
```

Comment:
What is the difference between `identifier` and `identifier.titleFromHostname` ? Is the latter just the full hostname without `https://` in front?

Reply:
Yes, indeed `identifier.titleFromHostname` is the hostname without the `https://` in front.